docs/sdk/py/client/index.mdx

```diff
@@ -0,0 +1,343 @@
+---
+title: Client Module
+description: API reference for the Client Module Module of the ReductStore Client SDK for Python.
+---
+<head>
+    <link rel="canonical" href="https://www.reduct.store/docs/sdk/py/client" />
+</head>
+
+# Client Module
+
+<a id="reduct.client"></a>
+
+# reduct.client
+
+Client module for ReductStore HTTP API
+
+<a id="reduct.client.Client"></a>
+
+## Client
+
+```python
+class Client()
+```
+
+HTTP Client for Reduct Storage HTTP API
+
+<a id="reduct.client.Client.__init__"></a>
+
+#### \_\_init\_\_
+
+```python
+def __init__(url: str,
+             api_token: Optional[str] = None,
+             timeout: Optional[float] = None,
+             extra_headers: Optional[Dict[str, str]] = None,
+             **kwargs)
+```
+
+Constructor
+
+**Arguments**:
+
+- `url` - URL to connect to the storage
+- `api_token` - API token if the storage uses it for authorization
+- `timeout` - total timeout for connection, request and response in seconds
+- `extra_headers` - extra headers to send with each request
+  Kwargs:
+- `session` - an external aiohttp session to use for requests
+- `verify_ssl` - verify SSL certificates
+
+**Example**:
+```python
+      client = Client("http://127.0.0.1:8383")
+      info = await client.info()
+   
+```
+
+<a id="reduct.client.Client.info"></a>
+
+#### info
+
+```python
+async def info() -> ServerInfo
+```
+
+Get high level server info
+
+**Returns**:
+
+  ServerInfo:
+  
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.list"></a>
+
+#### list
+
+```python
+async def list() -> List[BucketInfo]
+```
+
+Return a list of all buckets on server
+
+**Returns**:
+
+  List[BucketInfo]
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.get_bucket"></a>
+
+#### get\_bucket
+
+```python
+async def get_bucket(name: str) -> Bucket
+```
+
+Load a bucket to work with
+
+**Arguments**:
+
+- `name` - name of the bucket
+
+**Returns**:
+
+  Bucket
+
+**Raises**:
+
+- `ReductError` - if there is an HTTP error
+
+<a id="reduct.client.Client.create_bucket"></a>
```

Comment:
it's a bit difficult to spot h4 tags on the website (for example "create_bucket" looks the same **Raises**).

Maybe it makes sense to add separators ?

---
#### _get\_bucket_

...

**Arguments**:

...

**Returns**:

...

**Raises**:

...

---
#### _create\_bucket_

...

**Arguments**:

...

**Returns**:

...

**Raises**:

...

Reply:
Unfortunately, it's given by `pydoc-markdown`.